src/src/rx_main.cpp

```diff
@@ -1546,6 +1555,14 @@ static void setupSerial1()
             Serial1.begin(19200, SERIAL_8N2, serial1RXpin, serial1TXpin, false);
             serial1IO = new SerialHoTT_TLM(SERIAL1_PROTOCOL_TX, SERIAL1_PROTOCOL_RX, serial1TXpin);
             break;
+        case PROTOCOL_SERIAL1_TRAMP:
+            Serial1.begin(9600, SERIAL_8N1, serial1RXpin, serial1TXpin, false);
+            serial1IO = new SerialTramp(SERIAL1_PROTOCOL_TX, SERIAL1_PROTOCOL_RX);
+            break;
+        case PROTOCOL_SERIAL1_SMARTAUDIO:
+            Serial1.begin(4800, SERIAL_8N2, UNDEF_PIN, serial1TXpin, false);
+            serial1IO = new SerialSmartAudio(SERIAL1_PROTOCOL_TX, SERIAL1_PROTOCOL_RX);
```

Comment:
SA particularly needs to be half duplex.  SA VTxs always reply with a packet after receiving a cmd.

TRAMP might as well also be changed to half duplex.